### PR TITLE
feat [193]: add basic support for mobile

### PR DESCRIPTION
**Pull request by timrbula**
Closes #

Closes #193 (did I create that issue correctly??) 
https://github.com/boomerang-io/roadmap/issues/193

#### Changelog

**New**

- Added basic mobile support for Flow marketing page
- Added HTML title

**Changed**

- Fixed aspect ratio on YT videos for all browsers
- Some code formatting

**Removed**

- nada

#### Testing / Reviewing

View the Netlify deploy preview or locally w/ a smaller screen or with DevTools mobile viewer. 

## What it should look like now

image

### CODE REVIEW

File: src/components/flow/Intro/index.jsx [hunk 1]
```diff
@@ -1,44 +1,51 @@
...
-          {videoIsLoading && <div className={styles.videoLoading}><Loading withOverlay={false} /></div>}
-          <iframe 
-            allowFullScreen
-            className={styles.youtubeVideo}
-            frameborder="0"
-            loading="lazy"
-            onLoad={() => setVideoIsLoading(false)}
-            src="https://www.youtube.com/embed/-lOninwMoz4?start=0"
-          />
+          <div className={styles.videoContainer}>
```
Comment (timrbula) [inline src/components/flow/Intro/index.jsx:34]: Had to add container here for it to work on safari w/ the hack using padding and absolute positioning as mentioned in the css comment

File: src/pages/flow.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React from 'react'
```
Comment (timrbula) [inline src/pages/flow.js:1]: Sorry about the formatting noise, but I used the formatting from the included .prettierrc file in the project

**Review by timrbula — COMMENTED** [review]
Add a few comments for yall on the changes that I ended up making. It was not as simple as I would have hoped. Why would I ever think that.

**Comment by tlawrie** [comment]
LGTM!

I have verified using the preview in my browser and using Window Resizer in Chrome to various mobile device sizes.

@LucasGrimauth @BenjaminRuby what are your thoughts?

**Review by tlawrie — APPROVED** [review]

**Comment by tlawrie** [comment]
Hey @BenjaminRuby or @LucasGrimauth do either of you want to approve (or reject)?

**Review by tlawrie — APPROVED** [review]

**Pull request merged**